### PR TITLE
Add Contributing.md and ISSUE_TEMPLATE.md and PULL_REQUEST_TEMPLATE

### PR DESCRIPTION
**Pull request by ms10398**
For #1112 

Added Contributing.md for new comers.

Add Issue Template and Pull Request Template so that standards are followed in development.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+
+# Contributing
+
+When contributing to this repository, please first discuss the change you wish to make via issue, or the official [slack channel](https://slack-ssl-openfoodfacts.herokuapp.com/).
+
+## Pull Request Process
+
+1. Ensure any install or build dependencies are removed before the end of the layer when doing a 
+   build.
+2. Check that there are no conflicts and your request passes [Travis](Travis-ci.org) build. Check the log of the pass test if it fails the build.
```
Comment (hangy) [inline CONTRIBUTING.md]: Can you make this an absolute link? This does not work as expected when rendered in Markdown.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+* Make sure your PR's description contains GitHub's special keyword references that automatically close the related issue when the PR is merged. (For more info click [here]( https://github.com/blog/1506-closing-issues-via-pull-requests))
+* When you make very very minor changes to a PR of yours (like for example fixing a failing Travis build or some small style corrections or minor changes requested by reviewers) make sure you squash your commits afterward so that you don't have an absurd number of commits for a very small fix. (Learn how to squash at [here](https://davidwalsh.name/squash-commits-git )
+* When you're submitting a PR for a UI-related issue, it would be really awesome if you add a screenshot of your change or a link to a deployment where it can be tested out along with your PR. It makes it very easy for the reviewers and you'll also get reviews quicker.
+
+### Feature Requests and Bug Reports
+* When you file a feature request or when you are submitting a bug report to the [issue tracker](https://github.com/openfoodfacts/openfoodfacts-server/issues), make sure you add steps to reproduce it. Especially if that bug is some weird/rare one.
+
+## Code of Conduct
```
Comment (hangy) [inline CONTRIBUTING.md]: There's a CoC in the Wiki: https://en.wiki.openfoodfacts.org/Code_of_conduct Maybe we should link to that instead?

File: .github/ISSUE_TEMPLATE.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+**Checklist**
+- [x] Bug Report - The issue you are reporting might be there already. Please search on the [issues tracker](https://github.com/openfoodfacts/openfoodfacts-server/issues) before creating or reporting a issue.
```
Comment (hangy) [inline .github/ISSUE_TEMPLATE.md]: I'm not sure what's correct here. Are the checkboxes to be pre-checked for the user? They are not for the PR template.

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 4]
```diff
@@ -0,0 +1,18 @@
...
+
+## Checklist
+ 
+Make sure you've done all the following (You can delete the checklist before submitting)
+ 
+ - [ ] Code is well documented
+ - [ ] Include unit tests for new functionality
+ - [ ] Code passes Travis builds in your branch
+ - [ ] If you have multiple commits please combine them into one commit by squashing them.
+ - [ ] Read and understood the contribution guidelines .
```
Comment (hangy) [inline .github/PULL_REQUEST_TEMPLATE.md]: Should this link to the contribution guidelines?

File: CONTRIBUTING.md (hunk 1 above)
Comment (ms10398) [reply CONTRIBUTING.md]: Yeah sorry I didnt noticed it will do so surely

File: CONTRIBUTING.md (hunk 2 above)
Comment (ms10398) [reply CONTRIBUTING.md]: Okay thats great I will link it to that.

File: .github/ISSUE_TEMPLATE.md (hunk 3 above)
Comment (ms10398) [reply .github/ISSUE_TEMPLATE.md]: Yes they need not to be prechecked the user has to precheck it doing so :+1:

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 4 above)
Comment (ms10398) [reply .github/PULL_REQUEST_TEMPLATE.md]: Yes I think we should link it to the contribution guidelines.

**Comment by ms10398** [comment]
Will do all the suggested changes in the subsequent commit in half hour

**Comment by ms10398** [comment]
@hangy Did the suggested changes

Have a look again at it

Rebased the commit

**Comment by ms10398** [comment]
@teolemon @stephanegigandet Have a look

**Pull request merged**